include/DDTrackCreatorALLEGRO.h

```diff
@@ -0,0 +1,110 @@
+/**
+ *  @file   DDMarlinPandora/include/DDTrackCreatorALLEGRO.h
+ * 
+ *  @brief  Header file for the ILD implementation of the track creator class.
+ * 
+ *  $Log: $
+ */
+
+#ifndef DDTRACK_CREATOR_ALLEGRO_H
+#define DDTRACK_CREATOR_ALLEGRO_H 1
+
+#include "DDTrackCreatorBase.h"
+
+//------------------------------------------------------------------------------------------------------------------------------------------
+
+/**
+ *  @brief  DDTrackCreatorALLEGRO class
+ */
+class DDTrackCreatorALLEGRO : public DDTrackCreatorBase
```

Comment:
Why not inherit from DDTrackCreatorCLIC and reduce the amount of copy-pasta?

Reply:
The functions re-used in the DDTrackCreatorALLEGRO are supposed to change for ALLEGRO. Currently only few things are disabled, but implementation of the (truth) tracks in the ALLEGRO is not finished yet.